### DISCUSSION TITLE
WFB-NG Data Transport Standard [Draft]

### DISCUSSION THREAD

**Discussion by svpcom** [svpcom/wfb-ng#245] [Ideas]
[WFB-NG Data Transport Standard [Draft]](https://github.com/svpcom/wifibroadcast/blob/wfb-ng-std-draft/doc/wfb-ng-std-draft.md)
[Reference implementation](https://github.com/svpcom/wifibroadcast/tree/wfb-ng-std-draft)

**Comment by svpcom** [comment]
Main difference from previous releases: Multiple links support out of box

Now you can setup multiple links with same keypair. MAC address now has format: `0x57, 0x42, 0xaa, 0xbb, 0xcc, 0xdd`, where the first two bytes are the protocol header (`'W'`,`'B'`), then three bytes - the link id and the last byte - the number of the stream inside the link.

**Comment by svpcom** [comment]
@Dronotique Could you try this version in case of multiple drones setup? Use `wfb-ng-std-draft` branch instead of master.

**Comment by InanixFR** [comment]
@svpcom just let me understand... This version allows multiple drones on one unique wifi card on the ground station? Is there enough bandwith for multi-video stream?

For the moment my setup is multi drone with for each on wifi card on ground station.
I'll try to do some tests since the end of the week

**Reply by svpcom** [reply]
Yes, it allows to control multiple drones from one card (if bandwidth allow that)
